packages/kusama/src/__snapshots__/assetHubKusama.multisig.proxy.e2e.test.ts.snap

```diff
@@ -0,0 +1,180 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`Kusama AssetHub Multisig Proxy > failure tests > 2-of-3 multisig as standard proxy (non-transfer) > events when Alice creates multisig 1`] = `
+[
+  {
+    "data": {
+      "approving": "HNZata7iMYWmk5RvZRTiAsSDhV8366zq2YGb3tLH5Upf74F",
+      "callHash": "(hash)",
+      "multisig": "EF9xmEeFv3nNVM3HyLAMTV5TU8jua5FRXCE116yfbbrZbCL",
```

Comment:
Because the multisig address depends on its creation block, this address will change every time this test runs in PET's CI; it is best to redact this (and all other such occurrences).